added-ide-content

anaghapamidi opened the pull request:
Hi @Pubs-MV request a review for the IDE content.

anaghapamidi committed:
added-ide-content

anaghapamidi committed:
updates-to-intellij-content

Pubs-MV commented on code-security/admin_guide/get-started/connect-your-repositories/add-intellij.adoc:
```diff
@@ -0,0 +1,53 @@
+[.task]
+== Add IntelliJ to Prisma Cloud Code Security
+
+Integrating IntelliJ with Prisma Cloud Code Security makes it possible for you to identify misconfigurations before you commit your code, and avoid pull requests that can potentially fail builds due to undetected misconfigurations. Using Checkov, a code analysis tool that scans Infrastructure-as-code (IaC) files from frameworks such as Terraform, CloudFormation, and Azure Resource Manager (ARM) o ...
```
Is that the issue "avoid pull requests that can potentially fail builds due to undetected misconfiguration"?
or is to help "get real-time scan results to help developers in the initial stages of development and quickly resolve issues early in the software lifecycle"

anaghapamidi replied:
The way I look at it absolve potencial crisis via "avoid pull requests that can potentially fail builds due to undetected misconfiguration" and with additional feature help of "immediate detection of misconfigurations and inline code fixes." Though this line summarizes everything "get real-time scan results to help developers in the initial stages of development and quickly resolve issues early in the software lifecycle"
I will rephrase this?

Pubs-MV commented on code-security/admin_guide/get-started/connect-your-repositories/add-intellij.adoc:
```diff
@@ -0,0 +1,53 @@
+Integrating IntelliJ with Prisma Cloud Code Security makes it possible for you to identify misconfigurations before you commit your code, and avoid pull requests that can potentially fail builds due to undetected misconfigurations. Using Checkov, a code analysis tool that scans Infrastructure-as-code (IaC) files from frameworks such as Terraform, CloudFormation, and Azure Resource Manager (ARM) o ...
+Apart from the scanning the common industry standard policies, Checkov also identifies custom policies that you can configure during <<_add-a-new-custom-policy-for-build-time-checks,custom build-time checks>>.
+
+Before you begin with IntelliJ integration, you must first <<_generate-access-keys,generate access keys>> and have https://www.python.org/downloads/[Python] version 3.7 or above installed, as Checkov runs on Python. Alternatively, https://docs.pipenv.org/[Pipenv] or https://www.docker.com/products/docker-desktop[Docker] is also recommended. A configuration of Checkov plugin on IntelliJ during the ...
```
Are we saying to use the Checkov plugin on IntelliJ and the IntelliJ integration for Code Security?

anaghapamidi replied:
Yes. Since it is Checkov CLI that actually scans the IaC files. And then when the IDEs are integrated with Prisma Cloud, the custom policies can also be scanned. The integration between IDE and PC is mainly to access the custom policies.

Pubs-MV committed:
Update add-intellij.adoc

Edited for style. Why do we say both IntelliJ IDEA and IntelliJ? Can you please take a look?

anaghapamidi committed:
Updated content before IntelliJ fix

anaghapamidi committed:
updates to IDE, March 08

anaghapamidi committed:
few more changes

Pubs-MV reviewed (approved):
thank you
